src/middleware.js

```diff
@@ -69,13 +69,16 @@ export default function wrapper(context) {
         if (headers) {
           for (const name in headers) {
             if ({}.hasOwnProperty.call(headers, name)) {
-              res.set(name, context.options.headers[name]);
+              res.setHeader(name, context.options.headers[name]);
             }
           }
         }
 
-        res.send(content);
-
+        if (res.send) {
+          res.send(content);
+        } else {
+          res.end(content);
+        }
```

Comment:
Need encapsulate to utils with comments - `getHeader`, `setHeader`, `send` with code like:
```js
function getHeader(res, header) {
  // Fastify
  if (res.getHeader) {
    return res.getHeader(header)
  }

  // Express
  return res.get(header)
}
```

Maybe fastify has something in res what allow to known what it is fastify

Reply:
@evilebottnawi I think `getHeader` and `setHeader` works with all frameworks as `res` extend [responseServer](https://nodejs.org/dist/latest/docs/api/http.html#http_class_http_serverresponse)
the difference is that `.send` is express only function ref: https://github.com/fastify/fastify/issues/319

Reply:
@EslamHiko hm, if is is supported why we change it?

Reply:
I was wrong about `getHeader and setHeader works with all frameworks` I thought they all extend `ServerResponse`, what do you mean by `is`?